web/src/main/webResources/WEB-INF/config-db/jndi.xml

```diff
@@ -62,14 +52,12 @@
         <entry key="Data Source" value-ref="jdbcDataSource"/>
         <entry key="Loose bbox" value="true"/>
         <entry key="Estimated extends" value="true"/>
-        <entry key="schema" value="${database.schema}" />
+        <entry key="schema" value="public" />
```

Comment:
why are we setting this to public?
can't we take this value from the jndi configuration?

Reply:
I guess we can add as build property, but the previous one used `database.schema` was not really defined anywhere, so was not working.

I don't think can be moved to the container JNDI configuration, as this bean seem required by Geotools also. I'm not an expert in this part, @jodygarnett do you know if this could be completely delegated to the container JNDI configuration. 

If not I can check at least to make it configurable in the build.

Reply:
GeoTools uses the JNDI name, and whatever database.schema that connection was configured with. See [docs](https://docs.geotools.org/stable/userguide/library/jdbc/datastore.html#using-jndi).

Reply:
Updated to remove the parameter and rely on the JNDI resource configuration.

Reply:
afaik schema needs to be defined on two locations dns?currentschema=aaa, but also on config-spring-geonework.xml, one is used by hibernate, other by upgrade scripts. see also #2889